src/main/java/org/sagebionetworks/bridge/spring/controllers/ParticipantController.java

```diff
@@ -602,6 +604,26 @@ public StatusMessage sendSmsMessageForWorker(@PathVariable String appId, @PathVa
         participantService.sendSmsMessage(app, userId, template);
         return new StatusMessage("Message sent.");
     }
+
+    @PostMapping("/v3/participants/emailRoster")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public StatusMessage getParticipantRosterForWorker() throws JsonProcessingException {
+        UserSession session = getAuthenticatedSession(WORKER);
```

Comment:
This wouldn't be called by a worker. This would be called by a researcher or study coordinator.